### PR TITLE
Update deploy-gpu-node-pool.md

### PR DESCRIPTION
**Pull request by guanghuthegreat**
Update the GPU instructions to add the part of the dismount the drivers before the mitigation driver installation.

### CODE REVIEW

**Review by JamesLiang29 — APPROVED** [review]

**Comment by sethmanheim** [comment]
#sign-off

File: AKS-Hybrid/deploy-gpu-node-pool.md [hunk 1]
```diff
@@ -60,11 +60,50 @@ To use GPUs in AKS Arc, make sure you installed the necessary GPU drivers before
...
+Unknown       3D Video Controller               PCI\VEN_10DE&DEV_1EB8&SUBSYS_12A210DE&REV_A1\4&32EEF88F&0&0000 
+Unknown       3D Video Controller               PCI\VEN_10DE&DEV_1EB8&SUBSYS_12A210DE&REV_A1\4&3569C1D3&0&0000 
+```
+
+### Step 4: download and install the NVIDIA mitigation driver
 
 The software might include components developed and owned by NVIDIA Corporation or its licensors. The use of these components is governed by the [NVIDIA end user license agreement](https://www.nvidia.com/content/DriverDownload-March2009/licence.php?lang=us).
 
-See the [NVIDIA data center documentation](https://docs.nvidia.com/datacenter/tesla/gpu-passthrough/) to download the NVIDIA mitigation driver. After downloading the driver, expand the archive and install the mitigation driver on each host machine.
+See the [NVIDIA data center documentation](https://docs.nvidia.com/datacenter/tesla/gpu-passthrough/) to download the NVIDIA mitigation driver. After downloading the driver, expand the archive and install the mitigation driver on each host machine. You can follow this PowerShell script to download the migitation driver and extact it:
```
Comment (Jak-MS) [inline AKS-Hybrid/deploy-gpu-node-pool.md:106]:
```suggestion
See the [NVIDIA data center documentation](https://docs.nvidia.com/datacenter/tesla/gpu-passthrough/) to download the NVIDIA mitigation driver. After downloading the driver, expand the archive and install the mitigation driver on each host machine. You can follow this PowerShell script to download the mitigation driver and extract it:
```
fixing typos

File: AKS-Hybrid/deploy-gpu-node-pool.md [hunk 2]
```diff
@@ -60,11 +60,50 @@ To use GPUs in AKS Arc, make sure you installed the necessary GPU drivers before
...
+```
+
+You should see the GPU devices appear in an error state as shown in this example output:
+
+```output
+Error       3D Video Controller                   PCI\VEN_10DE&DEV_1EB8&SUBSYS_12A210DE&REV_A1\4&32EEF88F&0&0000 
+Error       3D Video Controller                   PCI\VEN_10DE&DEV_1EB8&SUBSYS_12A210DE&REV_A1\4&3569C1D3&0&0000 
+```
+
+### Step 3: Dismount the host driver from the host
```
Comment (Jak-MS) [inline AKS-Hybrid/deploy-gpu-node-pool.md:78]:
```suggestion
### Step 3: dismount the host driver from the host
```

File: AKS-Hybrid/deploy-gpu-node-pool.md [hunk 3]
```diff
@@ -60,11 +60,50 @@ To use GPUs in AKS Arc, make sure you installed the necessary GPU drivers before
 
 Install the Azure Stack HCI, version 23H2 operating system locally on each server in your Azure Stack HCI cluster.
 
-### Step 2: download and install the NVIDIA mitigation driver
+### Step 2: Uninstall the NVIDIA host driver
```
Comment (Jak-MS) [inline AKS-Hybrid/deploy-gpu-node-pool.md:63]:
```suggestion
### Step 2: uninstall the NVIDIA host driver
```

**Comment by Karl-WE** [comment]
@sethmanheim @Jak-MS may I ask about the state of my comment on this PR now that is is signed off and merged.

Shall I raise a new PR? Thank you for your time.

**Comment by Jak-MS** [comment]
@Karl-WE - if you had further suggested changes, please include them in a new PR. thank you!

**Comment by sethmanheim** [comment]
> @sethmanheim @Jak-MS may I ask about the state of my comment on this PR now that is is signed off and merged.
> 
> Shall I raise a new PR? Thank you for your time.

@Karl-WE Not sure what comment you might be referring to?

**Review by Karl-WE — COMMENTED** [review]
Mixed Code of PowerShell and cmd in line 68

**Comment by Karl-WE** [comment]
@sethmanheim @Jak-MS pardon me I commented on line 68 but didn't submit.

So rather new PR?

**Pull request merged**